src/lib/validation/utils/validateDatepicker.js

```diff
@@ -2,11 +2,19 @@ import { toggleErrorClass } from './helpers'
 
 export default function validateDatepicker (input) {
   let isValid = true
+
   if (input.value === '') {
-    isValid = false
+    isValid = !input.hasAttribute('required')
```

Comment:
I don't understand this logic. for me this means, setting isValid to true for all instances where no input is required. But, i can also input invalid data into a non-required field!

Reply:
it is about the empty string, whether the empty string is allowed or not. The problem is that we have to validate the Datepicker every time the user enters smth, not only if this field is required. Similar to the email field, it should be validated every time because there is a pattern. In this case, we can allow an empty string, like [here](https://github.com/demos-europe/demosplan-ui/blob/main/src/lib/validation/utils/validateInputField.js#L36)